docs/getting-started/developers/index.md

```diff
@@ -56,23 +56,42 @@ When setting up Warp in your project, you can choose to create an `uno.config` f
 
 - **Alternative 1: Add a uno.config file**
 
-Create a `uno.config.[js,ts,mjs,mts]` file with the following content. This file will configure UnoCSS with our Warp preset, including a safelist of component classes, which will add styling to Warp components. See all configuration options for `presetWarp` in the [Warp CSS docs](https://warp-ds.github.io/css-docs/plugin-api).
+Create a `uno.config.[js,ts,mjs,mts]` file with the following content. This file will configure UnoCSS with our Warp preset. See all configuration options for `presetWarp` in the [Warp CSS docs](https://warp-ds.github.io/css-docs/plugin-api).
 
 > uno.config.js
 
 ```js
 import { defineConfig } from 'unocss';
 import { presetWarp } from '@warp-ds/uno';
-import { classes } from '@warp-ds/component-classes/classes';
 
 export default defineConfig({
   presets: [presetWarp()],
-  safelist: classes,
 });
 ```
 
 By default, UnoCSS will automatically look in the root directory of your project for `uno.config.[js,ts,mjs,mts]` or `unocss.config.[js,ts,mjs,mts]`.
 
+#### Components
+
+If you are using Warp components there are two additional steps to do.
+
+1. You'll need to add `https://assets.finn.no/pkg/@warp-ds/css/1.0.0-alpha.10/components.css` in the head of your app or in the template if you are using such. This stylesheet is needed to add styling to Warp components.
```

Comment:
In line 32 and 38 we also instruct users to install component-classes, which can be removed with this change in the setup, I think.

Reply:
Ah yes, that needs to be removed. Good catch. I fix